src/ParallelAlgorithms/Amr/Actions/EvaluateRefinementCriteria.hpp

```diff
@@ -97,6 +100,17 @@ struct EvaluateRefinementCriteria {
       }
     }
 
+    // An element cannot join if it is splitting in another dimension
+    if (alg::any_of(overall_decision, [](amr::domain::Flag flag) {
+          return flag == amr::domain::Flag::Split;
+        })) {
+      for (size_t d = 0; d < volume_dim; ++d) {
+        if (gsl::at(overall_decision, d) == amr::domain::Flag::Join) {
+          gsl::at(overall_decision, d) = amr::domain::Flag::DoNothing;
```

Comment:
This overrides the priority of flags returned by the AMR criteria above: we select "do nothing" even though another criterion may have selected a lower-priority flag than "join" that's compatible with the split in the other dimension. Is that ok / expected behavior?

Reply:
there is no lower-priority flag than `Join` so this is okay.  I can add a comment if you'd like

Reply:
Sounds good. A comment would be helpful.

Reply:
okay, pushed a fixup